libs/sdk-ui-kit/src/Header/WorkspacePickerHomeFooter.tsx

```diff
@@ -4,6 +4,21 @@ import React from "react";
 /**
  * @internal
  */
-export const WorkspacePickerHomeFooter: React.FC = ({ children }) => {
-    return <div className="home-footer-icon">{children}</div>;
+export interface IWorkspacePickerHomeFooterProps {
+    href: string;
+    className?: string;
+}
+/**
+ * @internal
+ */
+export const WorkspacePickerHomeFooter: React.FC<IWorkspacePickerHomeFooterProps> = ({
+    children,
+    className,
+    href,
+}) => {
+    return (
+        <a className={`gd-workspace-picker-home-footer ${className || ""}`} href={href}>
```

Comment:
I think it would be cleaner to use cx from classnames to combine the classes here. It should handle the fallback for you.